types/src/v0/impls/stake_table.rs

```diff
@@ -126,10 +126,16 @@ struct Committee {
     /// leader but without voting rights.
     eligible_leaders: Vec<StakeTableEntry<PubKey>>,
 
-    /// TODO: add comment
+    /// Stake table
+    stake_table: Vec<StakeTableEntry<PubKey>>,
+
+    /// DA members
+    da_members: Vec<StakeTableEntry<PubKey>>,
+
+    /// Stake table indexed by public key, for efficient lookup.
     indexed_stake_table: BTreeMap<PubKey, StakeTableEntry<PubKey>>,
 
-    /// TODO: comment
+    /// DA members indexed by public key, for efficient lookup.
     indexed_da_members: BTreeMap<PubKey, StakeTableEntry<PubKey>>,
```

Comment:
Nit: maybe make the value here `usize` index into `stake_table` and `da_members`, so we only have a single source of truth for the actual contents of the stake table entry?

Reply:
Also consider `BTreeMap` -> `HashMap` so it is more clear that nothing is actually ordered by keys?